auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue

```diff
@@ -0,0 +1,506 @@
+                    <v-list-item-title>Use the business {{passwordText}}</v-list-item-title>
+                  </template>
+                  <!-- Passcode -->
+                  <div class="item-content">
+                    <v-text-field
+                      v-if="isBusinessIdentifierValid"
+                      filled
+                      :label="passcodeLabel"
+                      :hint="passcodeHint"
+                      persistent-hint
+                      :rules="passcodeRules"
+                      :maxlength="passcodeMaxLength"
+                      v-model="passcode"
+                      autocomplete="off"
+                      class="passcode mt-0 mb-2"
+                      :aria-label="passcodeLabel"
+                    />
+                    <!-- Certify (firms only) -->
+                    <Certify
+                      v-if="isBusinessIdentifierValid && isFirm"
+                      :certifiedBy="certifiedBy"
+                      entity="registered entity"
+                      @update:isCertified="isCertified = $event"
+                      class="certify"
+                      :class="(isBusinessIdentifierValid && showAuthorization) ? 'mt-4 mb-5' : 'mt-6 mb-5'"
+                    />
+                    <!-- Folio Number -->
+                    <section v-if="isBusinessIdentifierValid" class="mt-3 mb-2">
+                      <header class="font-weight-bold">Folio / Reference Number</header>
+                      <p class="mt-2 mb-0">
+                        If you file forms for a number of companies, you may want to enter a
+                        folio or reference number to help you keep track of your transactions.
+                      </p>
+                      <v-text-field
+                        filled hide-details
+                        label="Folio or Reference Number (Optional)"
+                        :maxlength="50"
+                        v-model="folioNumber"
+                        class="folio-number mt-4 mb-4"
+                        aria-label="Folio or Reference Number (Optional)"
+                      />
+                    </section>
+                  </div>
+                </v-list-group>
+
+                <v-list-group v-model="emailOption">
+                  <template v-slot:activator>
+                    <v-list-item-title>
+                      Confirm authorization using your registered office email address
+                      <div class="subtitle"> (If you forgot or don't have a business {{passwordText}})</div>
+                    </v-list-item-title>
+                  </template>
+                  <div style="color:#313132;">
+                    <div>
+                      An email will be sent to the registered office contact email of the business:
+                    </div>
+                    <div><b>{}**@********</b></div>
+                    <div style="margin:6px 5px 16px 0 !important">
```

Comment:
I see a lot of hard-coded styles in this template, which should use classnames instead.

Also it looks like line 92 is for debugging -- delete now or remember to delete it later :)